CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 [Unreleased]: https://github.com/atomist-rugs/rug-editors/compare/0.11.0...HEAD
```

Comment:
Are you planning to cut a release after this is merged? Might be good to prep the CHANGELOG for that.